datahub-web/@datahub/data-models/addon/entity/dataset/dataset-entity.ts

```diff
@@ -148,33 +148,33 @@ export class DatasetEntity extends BaseEntity<Com.Linkedin.Dataset.Dataset> {
   /**
    * Whether or not the dataset has been deprecated
    */
-  @alias('entity.dataset.deprecated')
+  @alias('entity.deprecated')
```

Comment:
Will this break the deprecation status page?

Reply:
This keeps parity with the mid-tier changes. Since before we were expecting a nested `dataset` property in our response that has all of the actual API information. Now this brings it closer to what our current internal logic for the API response looks like. I tested the dataset deprecation page as well as it seems to work. 

<img width="1166" alt="Screen Shot 2020-09-14 at 6 13 58 PM" src="https://user-images.githubusercontent.com/16459843/93153426-141ac180-f6b6-11ea-9de8-517fe0959495.png">